java/client/src/main/java/glide/api/commands/HyperLogLogBaseCommands.java

```diff
@@ -70,4 +70,25 @@ public interface HyperLogLogBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> pfcount(String[] keys);
+
+    /**
+     * Merges multiple HyperLogLog values into a unique value.<br>
+     * If the destination variable exists, it is treated as one of the source HyperLogLog data sets,
+     * otherwise a new HyperLogLog is created.
+     *
+     * @see <a href="https://redis.io/commands/pfmerge/">redis.io</a> for details.
+     * @param destKey Identifier of the destination HyperLogLog where the merged data sets will be
+     *     stored.
+     * @param sourceKeys The source HyperLogLog to merge.
+     * @return <code>OK</code>
+     * @example
+     *     <pre>{@code
+     * String response = client.pfmerge("new_HLL", "old_HLL_1", "old_HLL_2").get();
+     * assert response.equals("OK"); // new HLL was created with merged content of old ones
```

Comment:
Would prefer HLL to not be abbreviated for clarity. Applies throughout.

Reply:
Fixed